src/commands/sandbox_server.cr

```diff
@@ -0,0 +1,24 @@
+module Mint
+  class Cli < Admiral::Command
+    class SandboxServer < Admiral::Command
+      define_help description: "Server for compiling sandbox applications"
+
+      define_flag host : String,
+        description: "Change the host the server binds to. (Default: 0.0.0.1)",
+        default: ENV["HOST"]? || "0.0.0.1",
```

Comment:
I'd use either `127.0.0.1` or `0.0.0.0`.

Reply:
Yes, that's supposed to be `0.0.0.0`